packages/react-components/src/components/gauge/converters/convertSeries.ts

```diff
@@ -119,13 +105,10 @@ export const convertSeries = ({
       color: 'inherit', //color of the center value
       rich: {
         value: {
-          fontWeight: 'bolder', //font weight of the value
```

Comment:
Why are we removing the echarts value and unit visuals for our own?

Reply:
because the designs needed the alarm state to be above the alarm value. 

since echarts was rendering the value+correct color on a canvas, it would've taken a bit of math to find the exact placement of the alarm state if we chose to not remove the value+unit

now we are in full control of all the text displayed, we only need to use echarts to render the actual gauge

Reply:
sounds good thanks!